File: integration_tests/source_apps/singleproject-fsharp-2.0/Controllers/TestsController.fs
namespace singleproject_fsharp_2._0.Controllers

open System
open System.Collections.Generic
open System.Linq
open System.Threading.Tasks
open Microsoft.AspNetCore.Mvc

[<Route("/")>]
type TestsController () =
    inherit Controller()

    [<HttpGet>]
    member this.Get() =
        "Hello World!"

File: integration_tests/apps/test-2.2/Controllers/TestsController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Mvc;

namespace Test_2_2.Controllers
{
    /// <summary>
    /// This controller implements all of the tests for this app.
    /// </summary>
    [Route("/")]
    public class TestsController : Controller
    {
        /// <summary>
        /// Basic serving test, verifies that the app is running by returning a well known
        /// string.
        /// </summary>
        [HttpGet]
        public string ServingTest()
        {
            return "Hello World!";
        }
    }
}
